content/en/blog/2021/wasm-progress/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Progress of Envoy and Istio WebAssembly Extensibility
+description: An update on Envoy and Istio WebAssembly based extensibility effort.
+publishdate: 2021-02-09
+attribution: "Pengyuan Bian (Google)"
+keywords: [wasm,extensibility,WebAssembly]
+---
+
+Back to Istio 1.5 release, Istio introduced [WebAssembly based extensibility](/blog/2020/wasm-announce/) support.
+Over the past year, Istio, Envoy, and Proxy-Wasm communities have continued the effort to make WebAssembly extensibility stable, reliable, and easy to adopt.
+In this blog post, we will walk through the updates in this area at Istio 1.9 release.
+
+## Proxy-Wasm (WebAssembly) support merged in upstream Envoy
+
+The initial development work of Envoy WebAssembly runtime happened at `envoy-wasm` fork.
+Over time, we collected great feedback from community early adopters, as well as Istio first class Wasm extensions development, which helped us to continuously improve the runtime.
+Since Istio 1.8, WebAssembly support was merged into the Envoy main tree. This is a significant milestone for Istio and Envoy Wasm efforts, since it indicates that:
```

Comment:
```suggestion
These improvements unblocked merging Wasm support directly into Envoy, allowing it to become part of all official Envoy releases (as of the Istio 1.8 release). This is a significant milestone for Istio and Envoy Wasm efforts, since it indicates that:
```